src/test/scala/centaur/FinalDirsSpec.scala

```diff
@@ -28,29 +30,38 @@ object FinalDirsSpec {
 class FinalDirsSpec extends FlatSpec with Matchers with ParallelTestExecution {
   import FinalDirsSpec._
 
-  "final Outputs on local backend" should "place files in output dir" in 
-    testFinalOutputs(OutputsDirLocalTest, "final_workflow_outputs_dir", LocalCheckFiles())
+  case class Backend(id: String, checkFiles: CheckFiles)
+  val Jes = Backend("Jes", JesCheckFiles(StorageOptions.getDefaultInstance.getService))
+  val Local = Backend("Local", LocalCheckFiles())
+
+  val cromwellBackends = CentaurCromwellClient.backends.get.supportedBackends.map(_.toLowerCase)
+
+  "final Outputs on local backend" should "place files in output dir" in
+    testFinalOutputs(OutputsDirLocalTest, "final_workflow_outputs_dir", Local)
 
   "final log dirs on local backend" should "place log files in output dir when requested" in 
-    testFinalOutputs(LogDirLocalTest, "final_workflow_log_dir", LocalCheckFiles())
+    testFinalOutputs(LogDirLocalTest, "final_workflow_log_dir", Local)
 
   "final call logs dir in local" should "place call files in output dir when requested" in 
-    testFinalOutputs(CallLogsDirLocalTest, "final_call_logs_dir", LocalCheckFiles())
+    testFinalOutputs(CallLogsDirLocalTest, "final_call_logs_dir", Local)
 
   "final Logs dir on jes backend" should "place log files in output dir when requested" in 
-    testFinalOutputs(LogDirJesTest, "final_workflow_log_dir", JesCheckFiles(StorageOptions.getDefaultInstance.getService))
+    testFinalOutputs(LogDirJesTest, "final_workflow_log_dir", Jes)
 
   "final outputs on jes backend" should "place files in output dir" in 
-    testFinalOutputs(OutputsDirJesTest, "final_workflow_outputs_dir", JesCheckFiles(StorageOptions.getDefaultInstance.getService))
+    testFinalOutputs(OutputsDirJesTest, "final_workflow_outputs_dir", Jes)
 
   "final Call logs dir in Jes" should "place call files in GCS dir when requested" in 
-    testFinalOutputs(CallLogsDirJesTest, "final_call_logs_dir", JesCheckFiles(StorageOptions.getDefaultInstance.getService))
+    testFinalOutputs(CallLogsDirJesTest, "final_call_logs_dir", Jes)
 
-  def testFinalOutputs(path: Path, option: String, backend: CheckFiles) = 
-    Workflow.fromPath(path) match {
-      case Valid(w) => TestFormulas.runFinalDirsWorkflow(w, option, backend).run.get
-      case Invalid(e) => fail(s"Could not read logs test:\n -${e.toList.mkString("\n-")}")
-    }
+  def testFinalOutputs(path: Path, option: String, backend: Backend) =
+    (Workflow.fromPath(path), cromwellBackends.contains(backend.id)) match {
 
+      case (Valid(w), true) =>
+        TestFormulas.runFinalDirsWorkflow(w, option, backend.checkFiles).run.get
 
+      case (Invalid(e), _) => fail(s"Could not read logs test:\n -${e.toList.mkString("\n-")}")
+
+      case (Valid(_), _) => Monad[Test].pure(()) //valid but not proper backend, ignore
```

Comment:
I think this will show the test as succeeded even though it didn't actually do anything. IMO it would be even better if we could have it `ignore`ed. IIRC we had kind of the same problem with `BackendNoDockerSpec`.